packages/zoe/src/contracts/coveredCall.js

```diff
@@ -7,64 +7,78 @@ import { makeZoeHelpers } from '../contractSupport';
 
 const rejectMsg = `The covered call option is expired.`;
 
-/** @typedef {import('../zoe').ContractFacet} ContractFacet */
+/**
+ * In a covered call, the owner of a digital asset sells a call
+ * option. A call option is the right to buy the digital asset at a
+ * pre-determined price, called the strike price. The call option has an expiry
+ * date, at which point the contract is cancelled.
+ *
+ * In this contract, the expiry date is represented by the deadline at
+ * which the offer escrowing the underlying assets is cancelled.
+ * Therefore, the proposal for the underlying assets must have an
+ * exit record with the key "afterDeadline".
+ *
+ * The invite that the creator of the covered call receives is the
+ * call option and has the following additional information in the
+ * extent of the invite:
+ * { expirationDate, timerAuthority, underlyingAsset, strikePrice }
+ *
+ * The initial proposal should be:
+ * {
+ *   give: { UnderlyingAsset: assetAmount },
+ *   want: { StrikePrice: priceAmount  },
+ *   exit: { afterDeadline: { deadline: time, timer: timer } },
+ * }
+ * The result of the initiali offer is { payout, outcome }, where payout will
+ * eventually contain the strikePrice, and outcome is an assayable invitation to
+ * buy the underlying. Since the contract provides assurance that the underlying
```

Comment:
"underlyingAsset"

Reply:
financial folk often just say "the underlying", but you're right this audience won't expect that.